app/views/ideas/_form.html.erb

```diff
@@ -46,11 +46,40 @@
     <% if current_user.admin? %>
       <div class="govuk-form-group">
         <%= form.label :assigned_user_id, class: "govuk-label" %>
-        <%= form.collection_select(:assigned_user_id, User.all.where(:admin => true), :id, :email, :prompt => true) %>
+        <%= form.collection_select(:assigned_user_id, User.all.where(:admin => true), :id, :email, {include_blank: 'Please select' }, { class: "govuk-select" }) %>
```

Comment:
Missed this in the earlier review, we should always use the modern symbol style:
Instead of `:admin => true` use `admin: true`
Rubocop will tell you about it but not in views